### ISSUE TITLE
Vignette(s)

### ISSUE THREAD

**Issue by jgabry** [stan-dev/posterior#42]
We don’t necessarily need it for the beta release, but just starting an issue to decide what content we should include in a vignette or several. I could imagine one long vignette or perhaps several smaller vignettes focused on separate things e.g. one about the formats and one about summaries and diagnostics, etc.). 

Tentative list (add more items by editing this comment):
 
- [x] Vignette demonstrating the different draws formats and manipulating them (e.g., subset, mutate, etc.)
- [x] Vignette demonstrating summaries and diagnostics
- [ ] Vignette explaining how to interpret and use MCSEs (see also #44)

**Comment by mjskay** [comment]
One more minor TODO for this issue:

- [ ] Once the vignette describing the different draws formats exists, add a reference to it in the `rvar` vignette here:
  https://github.com/stan-dev/posterior/blob/0184c56f0f6f7a7f9183f418983c1d028fef2942/vignettes/rvar.Rmd#L99

**Comment by jgabry** [comment]
I'm going to start working on vignettes this week I hope.

**Comment by jgabry** [comment]
> One more minor TODO for this issue:
> 
> * [ ]  Once the vignette describing the different draws formats exists, add a reference to it in the `rvar` vignette here:
>   https://github.com/stan-dev/posterior/blob/0184c56f0f6f7a7f9183f418983c1d028fef2942/vignettes/rvar.Rmd#L99

Yeah we should also link the other way too (from the main overview vignette to the rvar vignette).  I'll take care of both of these in the PR with the main overview vignette that I should have ready soon. We also need to make sure the pkgdown site includes the vignettes, which I'll also do as part of the PR.
